apps/website/src/pages/docs/_layout.astro

```diff
@@ -17,6 +17,10 @@ const currentPage = new URL(Astro.request.url).pathname;
 </Global>
 
 <style lang='scss'>
+  :root {
+    --_content-width: min(100%, 70ch);
```

Comment:
If this is set on root and changed in other files, maybe it should not be prefixed by `_`.

There is also this convention of creating two layers of variables such that it can be controlled both from the outside and the inside. Might make sense here. See https://lea.verou.me/2021/10/custom-properties-with-defaults/#:~:text=pseudo%2Dprivate%20custom%20properties

Reply:
yes

Reply:
Added

Reply:
so now you can move it back down, it's no longer required in `:root` in this file

in `components.astro` it can be set anywhere so `:root` is fine there